pkg/controller/miganalytic/validation.go

```diff
@@ -30,6 +30,7 @@ const (
 	NotFound     = "NotFound"
 	KeyError     = "KeyError"
 	TestFailed   = "TestFailed"
+	NotReady     = "NotReady"
```

Comment:
Doesn't look like this is being consumed by MigAnalytic. The consumer is actually MigPlan. Therefore, this should be removed.